Assets/Controllers/GameManager.cs

```diff
@@ -125,11 +141,21 @@ public void ExecuteMoveForSelectedPawn()
             Debug.LogWarning("Trying to move a pawn when we don't have one selected in the game manager");
```

Comment:
Can this be removed or put behind a development environment setting of some kind? My lack of Unity framework experience isn't really helpful on tell you the how of doing that but generally you want to remove explicit logger statements and replace that kind of error trapping with tests.